src/components/LinearProgressBar/LinearProgressBar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { 
+  LinearProgressBar as CoreLinearProgressBar,
+  LinearProgressBarProps as CoreLinearProgressBarProps 
+  } from 'wix-ui-core/LinearProgressBar';
+import ToggleOn from 'wix-ui-icons-common/system/ToggleOn';
+import FormFieldError from 'wix-ui-icons-common/system/FormFieldError';
+import style from './LinearProgressBar.st.css';
+import { Tooltip } from '../Tooltip';
+import { bool } from 'prop-types';
+
+export interface LinearProgressBarProps extends CoreLinearProgressBarProps{
+  errorMessage?: string;
+  light?: boolean;
+}
+
+export const LinearProgressBar: React.SFC<LinearProgressBarProps> = (props: LinearProgressBarProps) => {
+
+  const { errorMessage, light, ...otherProps } = props;
+
+  return (
+    <CoreLinearProgressBar 
+      data-hook="progress-bar"
+      {...style('root', { light: light })}
+      {...otherProps}
+      successIcon={<ToggleOn />}
+      errorIcon={(
+        <Tooltip data-hook="tooltip" placement="top" content={errorMessage}>
+          <FormFieldError />
+        </Tooltip>)}
+    />
+  )
+}
+
```

Comment:
Still missing
```
static propTypes = {
   errorMessage: string,
   light: bool
};
```

Reply:
Still waiting with it for Sunday, as you made some changes on Thursday which I would want to discuss. All properties are exposed from UI Core, and I'm not sure it's what we want. I thought that only value, error, errorMessage, and light should be exposed. showProgressIndication, errorIcon, successIcon are currently exposed and I'm not sure it's a good thing